app/components/tests/app_spec.js

```diff
@@ -0,0 +1,74 @@
+// See this for some tips with Meteor: https://medium.com/all-about-meteorjs/unit-test-react-components-in-meteor-a19d96684d7d
+
+// If you import ../App you'll have to stub out Meteor, this is
+// why it's important to use controller-views that just setup
+// data and then the children can be very easily tested with
+// just props and state. We'll use a local component for an example
+
+import React from 'react/addons';
+//import App from '../App';
+//import $ from 'jquery'; // you could use jq to make life easier
+const TestUtils = React.addons.TestUtils;
+const Simulate = TestUtils.Simulate;
+
+// these should go into a spec helper module
+
+function renderComponent(comp, props) {
+  return TestUtils.renderIntoDocument(
+    React.createElement(comp, props)
+  );
+}
+
+function simulateClickOn(selector) {
+  var button = this.$el.find(selector)[0];
+  React.addons.TestUtils.Simulate.click(button);
+}
+
+
+const Post = React.createClass({
+  getDefaultProps() {
+    return {title: "Default Post Name"};
+  },
+  getInitialState() {
+    return { isVisible: true };
+  },
+  handleHide() {
+    this.setState({isVisible: false});
+  },
+  render() {
+    let visibleClass = (this.state.isVisible) ? 'block' : 'hidden';
+    return (
+      <div className='Post' style={{display: visibleClass }}>
+        <h1>{this.props.title}</h1>
+        <article>
+          How now brown cow
+        </article>
+        <button onClick={this.handleHide}>Hide</button>
+      </div>
+    );
+  }
+});
+
```

Comment:
Ah, any way we could make the test have something to do with what's actually in the App stub instead of this?